package.json

```diff
@@ -113,6 +114,7 @@
     "selenium-webdriver": "^4.0.0-alpha.1",
     "standard-version": "^5.0.2",
     "tib": "^0.4.0",
+    "typescript": "^3.3.4000",
```

Comment:
typescript 3.4.1 has just been released , feel free to upgrade it for the PR ;)

Reply:
Done!